<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="GLWidget" name="openGLWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_Save_State">
     <property name="title">
      <string>&amp;Save State</string>
     </property>
     <addaction name="action_Save_Slot_1"/>
     <addaction name="action_Save_Slot_2"/>
     <addaction name="action_Save_Slot_3"/>
    </widget>
    <widget class="QMenu" name="menu_Load_State">
     <property name="title">
      <string>&amp;Load State</string>
     </property>
     <addaction name="action_Load_Slot_1"/>
     <addaction name="action_Load_Slot_2"/>
     <addaction name="action_Load_Slot_3"/>
    </widget>
    <addaction name="action_Load_ROM"/>
    <addaction name="action_Open_Recent"/>
    <addaction name="separator"/>
    <addaction name="menu_Save_State"/>
    <addaction name="menu_Load_State"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Options">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="action_Configuration"/>
    <addaction name="separator"/>
    <addaction name="action_Video_Settings"/>
    <addaction name="action_Audio_Settings"/>
    <addaction name="action_Input_Settings"/>
   </widget>
   <widget class="QMenu" name="menuEmulator">
    <property name="title">
     <string>&amp;Emulator</string>
    </property>
    <addaction name="action_Pause"/>
    <addaction name="action_Stop"/>
    <addaction name="action_Reset"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuEmulator"/>
   <addaction name="menu_Options"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Load_ROM">
   <property name="text">
    <string>Open ROM...</string>
   </property>
   <property name="toolTip">
    <string>Open Chip-8 ROM</string>
   </property>
  </action>
  <action name="action_Open_Recent">
   <property name="text">
    <string>Open &amp;Recent ROM</string>
   </property>
   <property name="toolTip">
    <string>Open Recent Chip-8 ROM</string>
   </property>
  </action>
  <action name="action_Configuration">
   <property name="text">
    <string>Configuration...</string>
   </property>
   <property name="toolTip">
    <string>Emulator Configuration</string>
   </property>
  </action>
  <action name="action_Video_Settings">
   <property name="text">
    <string>Video Settings...</string>
   </property>
   <property name="toolTip">
    <string>Emulator Video Settings</string>
   </property>
  </action>
  <action name="action_Audio_Settings">
   <property name="text">
    <string>Audio Settings...</string>
   </property>
   <property name="toolTip">
    <string>Emulator Audio Settings</string>
   </property>
  </action>
  <action name="action_Input_Settings">
   <property name="text">
    <string>Input Settings...</string>
   </property>
   <property name="toolTip">
    <string>Emulator Input Settings</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit Emulator</string>
   </property>
  </action>
  <action name="action_Pause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause Emulator</string>
   </property>
  </action>
  <action name="action_Reset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset Emulator</string>
   </property>
  </action>
  <action name="action_Save_Slot_1">
   <property name="text">
    <string>Slot 1</string>
   </property>
   <property name="toolTip">
    <string>Save In Slot 1</string>
   </property>
  </action>
  <action name="action_Save_Slot_2">
   <property name="text">
    <string>Slot 2</string>
   </property>
   <property name="toolTip">
    <string>Save In Slot 2</string>
   </property>
  </action>
  <action name="action_Save_Slot_3">
   <property name="text">
    <string>Slot 3</string>
   </property>
   <property name="toolTip">
    <string>Save In Slot 3</string>
   </property>
  </action>
  <action name="action_Load_Slot_1">
   <property name="text">
    <string>Slot 1</string>
   </property>
   <property name="toolTip">
    <string>Load Save From Slot 1</string>
   </property>
  </action>
  <action name="action_Load_Slot_2">
   <property name="text">
    <string>Slot 2</string>
   </property>
   <property name="toolTip">
    <string>Load Save From Slot 2</string>
   </property>
  </action>
  <action name="action_Load_Slot_3">
   <property name="text">
    <string>Slot 3</string>
   </property>
   <property name="toolTip">
    <string>Load Save From Slot 3</string>
   </property>
  </action>
  <action name="action_Stop">
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop Emulator</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>IO/GLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
